### ISSUE TITLE
Split av_result into two fields

### ISSUE THREAD

**Issue by DC3-DCCI** [oasis-tcs/cti-stix2#213]
Right now the `av_result` field can store either the result / name assigned by the tool that ran the malware analysis or an entry from `malware-av-result-ov`.  I believe that we should specify that the `av_result` field is for `malware-av-result-ov` only with a new field being added for a vendor specific name.  In this case I suggest `av_result_name`.

This will allow parsers to detect a standard categorization based on the ov without losing out on specific vendor information which is valuable for end users as these can suggest malware family names.

The following is a sample of what this change would look like.

```
{
        "type": "malware-analysis",
        "spec_version": "2.1",
        "modified": "2020-01-17T12:16:36.520Z",
        "created": "2020-01-17T12:16:36.520Z",
        "created_by_ref": "identity--a463ffb3-1bd9-4d94-b02d-74e4f1658283",
        "id": "malware-analysis-0f4164d1-6baa-43bf-923f-8efd7b7e11a2",
        "product": "Fake AV Product",
        "av_result": "malicious",
        "configuration_version": "1.1.1.1",
        "av_result_name": "AV Specific Name",
        "analysis_started": "2018-05-13 23:51:24",
        "analysis_ended": "2018-05-13 23:51:24"
}
```

**Comment by jordan2175** [comment]
This is very similar to the problem we had in 2.0 with the labels property being overloaded with the type of thing, like indicator type.  So I would be in favor of making this change.  We will discuss on the next working call.

**Comment by allant0** [comment]
Seems reasonable but we should be clear on when only one is provided or are both required?

**Comment by jordan2175** [comment]
We talked about this on the 2020-01-28 working call.  The consensus on the call is to make the change. Ivan suggested that we make the av_result more general and just called it result or conclusion or something.  This way it works for static analysis as well.

**Comment by jordan2175** [comment]
Renamed the av_result property to just result per the working call.  We added the property result_name per the working call.  We also fixed the type for the "result" property that was "string" and changed it to "open-vocab" to be consistent with everything else in the document now that it is only one thing.  We also renamed the vocab to drop the "av" from the name. The new description for "result" is:

> The classification result as determined by the scanner or tool analysis process.
>
> The value for this property SHOULD come from the malware-result-ov open vocabulary.

The description for "result_name" is:

> The classification result or name assigned to the malware instance by the scanner tool.


We also fixed the description in the malware-result-ov to reflect these changes.
